### ISSUE TITLE
content: Update LH a11y audio and video audit posts

### ISSUE THREAD

**Issue by mfriesenhahn** [GoogleChrome/web.dev#1701]
**Author**: @mfriesenhahn
**Reviewer**: @robdodson 

Part of #1687 (Lighthouse a11y cleanup).

**Process**

- [x] 1. I've done my proposal: N/A
- [x] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc
- [x] 4. Draft approved — _reviewer checks this_
- [x] 5. I've submitted a pull request #1700
- [ ] 6. Pull request approved — _reviewer checks this_

**Comment by mfriesenhahn** [comment]
On hold pending:
- Response from Dylan @ Deque about why audits aren't failing. (Fail state needed for screenshots.) @robdodson following up.
- Further investigation of whether the `<track kind="descriptions">` element is the best way to provide audio descriptions. (See https://terrillthompson.com/1173 for reference.)

**Comment by robdodson** [comment]
So based on our discussion, I think the audio/video description and caption rules in axe currently generate a "needs review" notification to the developer. This isn't something that lighthouse will be able to surface. [See related bug](https://github.com/dequelabs/axe-core/issues/1855). I suggest moving audio/video caption/description stuff to manual audits.

**Comment by robdodson** [comment]
Also we should probably turn these audits off in LH cc @brendankenny

**Comment by mfriesenhahn** [comment]
Updates are _preliminarily_ complete, pending confirmation from LH team that media audits are being changed to manual. DO NOT MERGE until they confirm.

**Comment by kaycebasques** [comment]
@kaycebasques to find out if these audits will be manual audits moving forward.

**Comment by kaycebasques** [comment]
I pinged the LH team for a confirmation.

**Comment by carolrivero** [comment]
Waiting for confirmation from the LH team and accessibility group.

**Comment by kaycebasques** [comment]
I'm going to Icebox this because the work seems blocked by web platform work.
